server/pulp/server/webservices/views/search.py

```diff
@@ -57,18 +72,19 @@ def get(self, request):
         :return:        HttpReponse containing a list of objects that were matched by the request
         :rtype:         django.http.HttpResponse
         """
-        query = dict(request.GET)
+        query, options = self._parse_args(request.GET)
         query['filters'] = json.loads(request.GET.get('filters'))
 
         fields = query.pop('field', '')
         if fields:
             query['fields'] = fields
 
-        return self._generate_response(query)
+        query['filters'] = query.pop('filters')
```

Comment:
I think that was a place holder for `query, options = self._parse_args(request.GET)` that was never removed.